tegel/src/components/divider/divider.tsx

```diff
@@ -0,0 +1,37 @@
+import { Component, Prop, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'sdds-divider',
+  styleUrl: 'divider-component.scss',
+  shadow: true,
+})
+export class Divider {
+  /** Direction of the Divider, horizontal if not specified. */
+  @Prop() direction: 'horizontal' | 'vertical' = 'horizontal';
+
+  /** Sets the divider to light version */
+  @Prop() light: boolean = false;
+
+  /** Sets the divider to dark version */
+  @Prop() dark: boolean = false;
```

Comment:
I would recommend changing this to an enum instead. That way they both can't be true/false at the same time.

Reply:
Ah yea, I like that:
```suggestion
 enum DividerColor {
  Light = 'light',
  Dark = 'dark',
}

Reply:
Yep! You could do it just one line also: 
```suggestion
  /** Sets the divider to light version */
  @Prop() type: 'light' | 'dark' = 'light'
```

Reply:
This would set the type to light by default

Reply:
Sure, same as direction.

Reply:
Yeah exactly 🚀